apps/ts-minbar-test-react-components/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import config from '@fluentui/scripts/config';
 import * as fs from 'fs-extra';
+import * as fsExtra from 'fs-extra';
```

Comment:
double import from same package, pls dedupe - we'll have auto fixable lint for that soon

Reply:
removed in [b961173](https://github.com/microsoft/fluentui/pull/21828/commits/b9611738e6121f6b609ad8b7cbc34b9d375d4b39)